README.md

```diff
@@ -1,2 +1,357 @@
+  - [AbdomenCT](#abdomenct)
+
+# Overview
+
+Overview about the results on the **Lits** Dataset.
+The best result from runs (Dice score) is reported.
+A more detailed analysis is given in the [experiments](#defining-the-basemodel) section.
+
+| Model                | Dice     |
+|----------------------|:--------:|
+| UNET                 |  0.9574  |
+| NnU-Net              |  0.9645  |
+| AttentionU-Net       |  0.9599  |
+| UNETR                |  0.9356  | 
+
+### References
+
+This repository adopts code from the following sources:
+
+- **UNet** ([paper](https://arxiv.org/pdf/1505.04597.pdf), [source code](https://docs.monai.io/en/stable/networks.html#unet))
+- **nnU-Net** ([paper](https://arxiv.org/abs/1809.10486), [source code](https://github.com/MIC-DKFZ/nnUNet))
+- **AttentionU-Net** ([paper](https://arxiv.org/abs/1804.03999), [source code](https://docs.monai.io/en/stable/networks.html#attentionunet))
+- **UNETR** ([paper](https://arxiv.org/abs/2103.10504), [source code](https://docs.monai.io/en/stable/networks.html#unetr))
+  
+# Repository Structure
+```
+Repository Standard_LiverLesion_Segmentation
+│   README.md
+│   requirements.txt    
+│   environment.yml
+│   .gitignore
+|   .pre-commit-config.yaml
+└─── experimental
+│   │   Overlay.ipynb
+│   │   Slice Paths Generator.ipynb
+│   │   cropping_dataset.ipynb
+│   │   main.ipynb
+│   
+└─── liver_imaging_analysis
+|   │   config
+|   |   │   configs.json
+|   |   engine
+|   |   │   config.py
+|   |   │   engine.py
+|   |   │   dataloader.py
+|   |   │   transforms.py
+|   |   │   utils.py
+|   |   |   visualization.py
+|   |   models
+|   |   │   lesion_segmentation.py
+|   |   │   liver_segmentation.py
+|   |   │   lobe_segmentation.py
+|   |   │   spleen_segmentation.py
+│       
+└─── tests
+|   │  test_config.py
+|   │  test_engine.py
+|   │  test_models.py
+|   │  test_transforms.py
+|   │  test_utils.py
+|   │  test_visualization.py
+|   |  test_dataloader.py
+│   
+└─── scripts
+
+```
+
+# How To Run
+
+## Requirements
+
+Install the needed packages by the following command. You might need to adapt the cuda versions for
+torch and torchvision specified in *requirements.txt*.
+Find a pytorch installation guide for your
+system [here](https://pytorch.org/get-started/locally/#start-locally).
+
+````shell
+pip install -r requirements.txt
+````
+
+Among others, this repository is mainly built on the following packages.
+You may want to familiarize yourself with their basic use beforehand.
+
+- **[Pytorch](https://pytorch.org/)**: The machine learning/deep learning library used in this
+  repository.
+- **[Black](https://black.readthedocs.io/en/stable/)**: Code Formatter used in this Repo. Check out the [official docs](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html) on how to use Black or [here](https://akshay-jain.medium.com/pycharm-black-with-formatting-on-auto-save-4797972cf5de) on how to integrate Black into PyCharm.
```

Comment:
Really! Black is the second most important package in this repo not MONAI, or optuna?

Reply:
i have handled

Reply:
please remove the Black package.